service/common/logevent/logevent.go

```diff
@@ -0,0 +1,16 @@
+package logevent
+
+import "github.com/indeedeng/iwf/gen/iwfidl"
+
+// The implementation must be lightweight, reliable and fast (less than 1s)
+type LogEventFunc func(event iwfidl.IwfEvent)
```

Comment:
Sorry for changing my mind on naming -- since this is event, maybe calling it "HandleEventFunc"?

because it may not be logging -- we may be sending to Kafka and consuming for DB etc

Reply:
Done